src/button/styles/button.css

```diff
@@ -0,0 +1,47 @@
+@import '../../styles/variables.css';
+
+.button {
+	background-color: var(--button-bg);
+	border: 1px solid var(--button-bg);
+	box-shadow: inset 0 0 0 1px rgba(255, 255, 255, 0.25);
+	border-radius: 5px;
+	color: var(--button-color);
+	cursor: pointer;
+	font: var(--base-font);
+	font-weight: bold;
+	display: inline-block;
+	padding: 0.75em 1.25em;
+	transition: all 0.3s ease;
+}
+
+.button:hover,
+.button:focus,
+.button[aria-pressed="true"] {
+	background-color: var(--button-hover-bg);
+	border-color: var(--button-hover-bg);
+	outline: none;
+}
+
+.button[disabled],
+.button[disabled]:hover {
+	background-color: var(--button-disabled-bg);
+	border-color: var(--button-disabled-bg);
+	cursor: default;
+}
+
+.button[data-dojo-icon]:before {
```

Comment:
can you use a class for this instead please? It feels to me that the icon should be passed as a property on the button widget itself rather than via a data-attr. Also worth noting that these rules around font size, weight etc... should be overridable by the icon-font used which may be font-awesome, material-icons etc...